### ISSUE TITLE
Cannot perform any action after login / after last update

### ISSUE THREAD

**Issue by ValJed** [hyprwm/Hyprland#8682]
### Already reported ? *

- [X] I have searched the existing open and closed issues.

### Regression?

Yes

### System Info and Version

<details> <summary>Nixos: 24.11.20241109.76612b1 (Vicuna)</summary>

```sh
Hyprland, built from branch v0.44.1-b at commit 4520b30d498daca8079365bdb909a8dea38e8d55  (version: bump to 0.44.1).
Date: 2024-10-09
Tag: v0.44.1, commits: 4520b30d498daca8079365bdb909a8dea38e8d55
built against aquamarine 0.4.3


flags: (if any)


System Information:
System name: Linux
Node name: jedxps
Release: 6.11.7
Version: #1-NixOS SMP PREEMPT_DYNAMIC Fri Nov  8 15:31:04 UTC 2024


GPU information: 
00:02.0 VGA compatible controller [0300]: Intel Corporation CoffeeLake-H GT2 [UHD Graphics 630] [8086:3e9b] (prog-if 00 [VGA controller])


os-release: ANSI_COLOR="1;34"
BUG_REPORT_URL="https://github.com/NixOS/nixpkgs/issues"
BUILD_ID="24.11.20241109.76612b1"
CPE_NAME="cpe:/o:nixos:nixos:24.11"
DEFAULT_HOSTNAME="jed_xps"
DOCUMENTATION_URL="https://nixos.org/learn.html"
HOME_URL="https://nixos.org/"
ID=nixos
ID_LIKE=""
IMAGE_ID=""
IMAGE_VERSION=""
LOGO="nix-snowflake"
NAME=NixOS
PRETTY_NAME="NixOS 24.11 (Vicuna)"
SUPPORT_URL="https://nixos.org/community.html"
VARIANT=""
VARIANT_ID=""
VENDOR_NAME=NixOS
VENDOR_URL="https://nixos.org/"
VERSION="24.11 (Vicuna)"
VERSION_CODENAME=vicuna
VERSION_ID="24.11"


plugins:

======Config-Start======


Config File: /home/jed/.config/hypr/hyprland.conf: Read Succeeded
source = ~/dotfiles/hypr/default.conf 

# See https://wiki.hyprland.org/Configuring/Monitors/
monitor=,preferred,auto,1.6


Config File: /home/jed/dotfiles/hypr/default.conf: Read Succeeded
###################
### MY PROGRAMS ###
###################

# See https://wiki.hyprland.org/Configuring/Keywords/

# Set programs that you use
$terminal = kitty
$fileManager = thunar
$menu = rofi -show drun -show-icons 
$menu-clipboard = cliphist list | rofi -dmenu | cliphist decode | wl-copy 

#################
### AUTOSTART ###
#################

# Autostart necessary processes (like notifications daemons, status bars, etc.)
# Or execute your favorite apps at launch like this:

# exec-once = $terminal
exec-once = wbg ~/dotfiles/assets/wallpaper.png
exec-once = dunst
exec-once = waybar
exec-once = nm-applet --indicator
exec-once = wl-paste --type text --watch cliphist store # Stores only text data
exec-once = wl-paste --type image --watch cliphist store # Stores only image data

#############################
### ENVIRONMENT VARIABLES ###
#############################

# See https://wiki.hyprland.org/Configuring/Environment-variables/
env = HYPRCURSOR_THEME,rose-pine-hyprcursor
env = XCURSOR_SIZE,24
env = HYPRCURSOR_SIZE,24

#####################
### LOOK AND FEEL ###
#####################

# Refer to https://wiki.hyprland.org/Configuring/Variables/

# https://wiki.hyprland.org/Configuring/Variables/#general
general { 
    gaps_in = 5
    gaps_out = 5
    border_size = 1

    # https://wiki.hyprland.org/Configuring/Variables/#variable-types for info about colors
    col.active_border = rgb(eb6f92)
    col.inactive_border = rgb(191724)

    # Set to true enable resizing windows by clicking and dragging on borders and gaps
    resize_on_border = false 

    # Please see https://wiki.hyprland.org/Configuring/Tearing/ before you turn this on
    allow_tearing = false

    layout = dwindle
}

# https://wiki.hyprland.org/Configuring/Variables/#decoration
decoration {
    rounding = 10

    # Change transparency of focused and unfocused windows
    active_opacity = 1.0
    inactive_opacity = 1.0

    # shadow = {
    #   enabled = true
    #   range = 4
    #   render_power = 3
    #   color = rgba(1a1a1aee)
    # }
 
    # https://wiki.hyprland.org/Configuring/Variables/#blur
    blur {
        enabled = true
        size = 3
        passes = 1
        
        vibrancy = 0.1696
    }
}

# https://wiki.hyprland.org/Configuring/Variables/#animations
animations {
    enabled = true

    # Default animations, see https://wiki.hyprland.org/Configuring/Animations/ for more

    bezier = myBezier, 0.05, 0.9, 0.1, 1.05

    animation = windows, 1, 5, myBezier
    animation = windowsOut, 1, 5, default, popin 80%
    animation = border, 1, 7, default
    animation = borderangle, 1, 7, default
    animation = fade, 1, 5, default
    animation = workspaces, 1, 5, default
}

# See https://wiki.hyprland.org/Configuring/Dwindle-Layout/ for more
dwindle {
    pseudotile = true # Master switch for pseudotiling. Enabling is bound to mainMod + P in the keybinds section below
    preserve_split = true # You probably want this
}

# See https://wiki.hyprland.org/Configuring/Master-Layout/ for more
master {
    new_status = master
}

# https://wiki.hyprland.org/Configuring/Variables/#misc
misc { 
    force_default_wallpaper = -1 # Set to 0 or 1 to disable the anime mascot wallpapers
    disable_hyprland_logo = false # If true disables the random hyprland logo / anime girl background. :(
}

#############
### INPUT ###
#############

# https://wiki.hyprland.org/Configuring/Variables/#input
input {
    kb_layout = fr
    kb_variant =
    kb_model =
    kb_options =
    kb_rules =

    follow_mouse = 1

    sensitivity = 0 # -1.0 - 1.0, 0 means no modification.

    touchpad {
        natural_scroll = false
    }
}

# https://wiki.hyprland.org/Configuring/Variables/#gestures
gestures {
    workspace_swipe = false
}

# Example per-device config
# See https://wiki.hyprland.org/Configuring/Keywords/#per-device-input-configs for more
device {
    name = epic-mouse-v1
    sensitivity = -0.5
}

####################
### KEYBINDINGSS ###
####################

# See https://wiki.hyprland.org/Configuring/Keywords/
$mainMod = ALT # Sets "ALT" key as main modifier

# Example binds, see https://wiki.hyprland.org/Configuring/Binds/ for more
bind = $mainMod, RETURN, exec, $terminal
bind = $mainMod, A, killactive,
bind = $mainMod SHIFT, M, exit,
bind = $mainMod, F, exec, $fileManager
bind = $mainMod, space, togglefloating,
bind = $mainMod, D, exec, $menu
bind = $mainMod, C, exec, $menu-clipboard
bind = $mainMod, S, exec, ~/dotfiles/scripts/rofi-sink.py
# bind = $mainMod, P, pseudo, # dwindle
bind = $mainMod, E, togglesplit, # dwindle
bind = $mainMod, Z, fullscreen, 1 
bind = $mainMod, Q, fullscreen, 
bind = $mainMod, ASTERISK, exec, hyprlock 

# Move focus
bind = $mainMod, H, movefocus, l
bind = $mainMod, L, movefocus, r
bind = $mainMod, K, movefocus, u
bind = $mainMod, J, movefocus, d

# Move windows
bind = $mainMod SHIFT, H, movewindow, l
bind = $mainMod SHIFT, J, movewindow, d
bind = $mainMod SHIFT, K, movewindow, u
bind = $mainMod SHIFT, L, movewindow, r

# Switch workspaces with mainMod + [0-9]
bind = $mainMod, ampersand, workspace, 1
bind = $mainMod, eacute, workspace, 2
bind = $mainMod, quotedbl, workspace, 3
bind = $mainMod, apostrophe, workspace, 4
bind = $mainMod, parenleft, workspace, 5
bind = $mainMod, minus, workspace, 6
bind = $mainMod, egrave, workspace, 7
bind = $mainMod, underscore, workspace, 8
bind = $mainMod, ccedilla, workspace, 9
bind = $mainMod, agrave, workspace, 10

# Move active window to a workspace with mainMod + SHIFT + [0-9]
bind = $mainMod SHIFT, ampersand, movetoworkspace, 1
bind = $mainMod SHIFT, eacute, movetoworkspace, 2
bind = $mainMod SHIFT, quotedbl, movetoworkspace, 3
bind = $mainMod SHIFT, apostrophe, movetoworkspace, 4
bind = $mainMod SHIFT, parenleft, movetoworkspace, 5
bind = $mainMod SHIFT, minus, movetoworkspace, 6
bind = $mainMod SHIFT, egrave, movetoworkspace, 7
bind = $mainMod SHIFT, underscore, movetoworkspace, 8
bind = $mainMod SHIFT, ccedilla, movetoworkspace, 9
bind = $mainMod SHIFT, agrave, movetoworkspace, 10

# Screenshot screen
bind = , PRINT, exec, grim - | swappy -f -

# Screenshot region
bind = SHIFT, PRINT, exec, grim -g "$(slurp -w 0)" - | wl-copy 
bind = CTRL SHIFT, PRINT, exec, grim -g "$(slurp -w 0)" - | swappy -f -

# Setup wf-recorder
# TODO

# Scroll through existing workspaces with mainMod + scroll
bind = $mainMod, mouse_down, workspace, e+1
bind = $mainMod, mouse_up, workspace, e-1

# Volume
bindle =, XF86AudioRaiseVolume, exec, wpctl set-volume @DEFAULT_AUDIO_SINK@ 5%+
bindle =, XF86AudioLowerVolume, exec, wpctl set-volume @DEFAULT_AUDIO_SINK@ 5%-
bindl =, XF86AudioMute, exec, wpctl set-mute @DEFAULT_AUDIO_SINK@ toggle

# Media Player
bind =, XF86AudioPlay, exec, playerctl play-pause
bind =, XF86AudioPause, exec, playerctl pause
bind =, XF86AudioNext, exec, playerctl next
bind =, XF86AudioPrev, exec, playerctl previous

# Luminosity
bindle = ,XF86MonBrightnessDown, exec, brightnessctl s 10%-
bindle = ,XF86MonBrightnessUp, exec, brightnessctl s +10%

# Move/resize windows with mainMod + LMB/RMB and dragging
bindm = $mainMod, mouse:272, movewindow
bindm = $mainMod, mouse:273, resizewindow

# Resize
bind = $mainMod, R, submap, resize

# will start a submap called "resize"
submap = resize

# # sets repeatable binds for resizing the active window
binde = , right, resizeactive, 30 0
binde = , left, resizeactive, -30 0
binde = , up, resizeactive, 0 -30
binde = , down, resizeactive, 0 30

# use reset to go back to the global submap
bind = , escape, submap, reset 

# will reset the submap, which will return to the global submap
submap = reset

##############################
### WINDOWS AND WORKSPACES ###
##############################

# See https://wiki.hyprland.org/Configuring/Window-Rules/ for more
# See https://wiki.hyprland.org/Configuring/Workspace-Rules/ for workspace rules

# Example windowrule v1
# windowrule = float, ^(kitty)$

# Example windowrule v2
# windowrulev2 = float,class:^(kitty)$,title:^(kitty)$

windowrulev2 = suppressevent maximize, class:.* # You'll probably like this.


======Config-End========
```

</details>

### Description

On NixOs, after the last update, I can log into hyprland but I cannot perform any action.
No config error are printed, cannot launch kitty (that is properly accessible when login through tty).

### How to reproduce

Can go on the last commit of my [dotfiles](https://github.com/ValJed/dotfiles/commit/0a7012d874a1df9257c1a36f54df1a1912550fd1)

### Crash reports, logs, images, videos

Logs:
```
[LOG] [AQ] Session timeout reached
[LOG] [AQ] Session could not be activated in time
[ERR] [AQ] DRM Backend failed
[LOG] [AQ] Starting the Aquamarine backend!
[LOG] [AQ] Starting the Wayland backend!
[ERR] [AQ] Wayland backend cannot start: wl_display_connect failed (is a wayland compositor running?)
[ERR] [AQ] Requested backend (wayland) could not start, enabling fallbacks
[ERR] [AQ] Implementation wayland failed, erasing.
[CRITICAL] m_pAqBackend couldn't start! This usually means aquamarine could not find a GPU or encountered some issues. Make sure you're running either on a tty or on a Wayland session, NOT an X11 one.
[CRITICAL] Critical error thrown: CBackend::create() failed!
```

**Comment by nicobonada** [comment]
I had the same problem (kitty not launching) and for me it was due to your commit here:  https://github.com/ValJed/dotfiles/commit/c9c57f364209cac64093b4f70affb9313c089961

I fixed it with `fc-cache -r`

**Comment by ValJed** [comment]
Oh wow, I would have never guessed that..
Gonna try that and closing for now.
Thank you!

**Comment by ValJed** [comment]
Reopening because I tried what @nicobonada suggested but it fixed nothing for me.
This is not specific to kitty because I'm not able to open any app like rofi or thunar.
I removed my hypr config to use the default one, didn't work better.
I even tried to update the config to use alacritty instead of kitty and wofi instead of rofi, same issue.

Fortunately I'm on Nix so I switched back again to a previous version, but yes last update just broke everything for me.

Not sure it's fully related to hyprland though.
